pkg/vulnsrc/suse-cvrf/suse-cvrf.go

```diff
@@ -44,27 +44,36 @@ var (
 	}
 )
 
+type PutInput struct {
+	Cvrf         SuseCvrf
+	Vuln         types.VulnerabilityDetail
+	AffectedPkgs []AffectedPackage
+}
+
 type DB interface {
 	db.Operation
-	Put(tx *bolt.Tx, cvrf SuseCvrf, vuln types.VulnerabilityDetail, affectedPkgs []AffectedPackage) error
+	Put(tx *bolt.Tx, input PutInput) error
```

Comment:
`alma`, `rocky` and `oracle-linux` have a `Get` function for the `DB` interface.
Don't you need this function?

Reply:
For suse, we want to use OSS Get function.